lldb/source/Commands/CommandObjectTarget.cpp

```diff
@@ -2691,6 +2691,7 @@ class CommandObjectTargetModulesDumpSeparateDebugInfoFiles
                     "Found unsupported debug info type '%s'.\n",
                     type.str().c_str());
               }
+              strm.EOL();
```

Comment:
This change looks unrelated? Why is this here?

Reply:
The default `image dump separate-debug-info` table output is a bit cramped without the extra space. I mentioned this in the PR summary 
> Also added a line of space in the default table output.
But I can move this change to a separate PR, sorry!